packages/fxa-payments-server/src/components/DialogMessage.tsx

```diff
@@ -2,10 +2,10 @@ import React, { ReactNode } from 'react';
 import classNames from 'classnames';
 import { useClickOutsideEffect } from '../lib/hooks';
 
-import './DialogMessage.scss';
-
 import Portal from './Portal';
 
+import './DialogMessage.scss';
+
```

Comment:
This fixes a `mini-css-extrac-plugin` warning that's treated as an error when the `CI` env var is set.  (See https://github.com/facebook/create-react-app/issues/5372)